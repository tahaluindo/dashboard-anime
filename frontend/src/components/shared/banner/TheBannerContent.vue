<template>
  <span
    class="absolute flex items-end justify-end w-1/2 text-white bottom-1/20 right-1/20 h-1/4"
  >
    <slot />
  </span>
</template>

<script lang="ts">
import { defineComponent } from "vue";

export default defineComponent({});
</script>

<style lang="postcss">
</style>