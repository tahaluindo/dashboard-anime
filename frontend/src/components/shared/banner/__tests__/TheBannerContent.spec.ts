import { mount } from "@vue/test-utils";
import TheBannerContent from "../TheBannerContent.vue";

describe("TheBannerContent component test", () => {
  it("should render the default component", () => {
    const wrapper = mount(TheBannerContent);

    expect(wrapper.html()).toMatchSnapshot();
  });
});
