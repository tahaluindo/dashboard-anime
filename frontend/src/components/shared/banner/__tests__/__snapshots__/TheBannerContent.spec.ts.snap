// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TheBannerContent component test should render the default component 1`] = `"<span class=\\"absolute flex items-end justify-end w-1/2 text-white bottom-1/20 right-1/20 h-1/4\\"></span>"`;
